modules/ocf_mirrors/manifests/projects/freebsd.pp

```diff
@@ -0,0 +1,25 @@
+class ocf_mirrors::projects::freebsd {
+  file {
+    '/opt/mirrors/project/freebsd':
+      ensure  => directory,
+      source  => 'puppet:///modules/ocf_mirrors/project/freebsd',
+      owner   => mirrors,
+      group   => mirrors,
+      mode    => '0755',
+      recurse => true;
+  }
+
+  ocf_mirrors::monitoring { 'freebsd':
+    type          => 'unix_timestamp',
+    upstream_host => 'mirror.math.princeton.edu',
```

Comment:
I think we should really be using the official upstream for this (`http://ftp.freebsd.org/pub/FreeBSD/TIMESTAMP`), instead of another mirror that could be out of date.